### PR TITLE
TELCODOCS-82: Cluster provisioning

### PR DESCRIPTION
**Pull request by StephenJamesSmith**
For version 4.8 Webscale Dev Preview

https://issues.redhat.com/browse/TELCODOCS-82

Direct link to doc preview:  https://deploy-preview-34819--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected

Signed-off-by: Stephen Smith stesmith@redhat.com

### CODE REVIEW

File: modules/ztp-performance-addon-operator.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+[id="ztp-performance-addon-operator_{context}"]
+= Performance Addon Operator
+
+The Performance Addon Operator provides the ability to enable advanced node performance tunings on a set of nodes.
+
+{product-title} (OCP) provides a Performance Addon Operator to implement automatic tuning to achieve low latency
+performance for OpenShift applications. The cluster administrator uses this performance profile configuration that makes
+it easier to make these changes in a more reliable way.
+The administrator can specify whether to update the kernel to rt-kernel, the CPUs to be reserved for
```
Comment (skrthomas) [inline modules/ztp-performance-addon-operator.adoc:13]: The list here needs to be more parallel. Suggest something like this, but up to you: The administrator can specify uptdating the kernel to rt-kernel, reserving CPUs for management workloads, and using CPUs for running the workloads.

File: modules/ztp-applying-the-ran-policies-for-monitoring-cluster-activity.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="ztp-applying-the-ran-policies-for-monitoring-cluster-activity_{context}"]
+= Applying the RAN policies for monitoring cluster activity
+
+Zero touch provisioning (ZTP) uses {rh-rhacm-first} as an orchestrator to apply the radio access network (RAN) policies using a policy-based governance approach to automatically monitor cluster activity.
```
Comment (ijolliffe) [inline modules/ztp-applying-the-ran-policies-for-monitoring-cluster-activity.adoc:8]: strike orchestrator (overloaded term)

File: modules/ztp-sriov-operator.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="ztp-sriov-operator_{context}"]
+= SR-IOV Operator
+
+The Single Root I/O Virtualization (SR-IOV) Network Operator manages the SR-IOV network devices and network attachments in your cluster.
+
+The SR-IOV Operator allows network interfaces to be virtualized and shared at a device level with networking functions running within the cluster.
```
Comment (skrthomas) [inline modules/ztp-sriov-operator.adoc:10]: Could virtualized just be virtual?

**Review by skrthomas — COMMENTED** [review]
One small comment and a question. Nice work :)

**Review by ijolliffe — CHANGES_REQUESTED** [review]
/lgtm would like the one minor comment addressed, but, not critical

**Comment by StephenJamesSmith** [comment]
@juphoff @ijolliffe  Can I get your /lgtm?

**Review by ijolliffe — COMMENTED** [review]
/lgtm

**Review by juphoff — APPROVED** [review]
/lgtm

**Comment by StephenJamesSmith** [comment]
@vikram-redhat Distros: openshift-webscale added back to topic map, Final assembly file included. Please merge.

**Comment by StephenJamesSmith** [comment]
Conflict resolved.

**Comment by StephenJamesSmith** [comment]
@vikram-redhat  Resolved conflict.

**Comment by bergerhoffer** [comment]
Commits/diff/preview looks correct to me now. This can be merged.

**Review by bobfuru — APPROVED** [review]

**Pull request merged**